src/main/java/project/controller/DriverRiderController.java

```diff
@@ -41,4 +42,37 @@ public DriverRiderController(UserService userModel, MockDataService mockDataServ
         return new ResponseEntity<DriverRiderResponse>(driverRiderResponse, HttpStatus.OK);
     }
 
+    @RequestMapping(value = "/registerrider", method = RequestMethod.POST)
+    public ResponseEntity<Object> registerRriver(
+            @RequestParam(value="phone_number") String phoneNumber,
+            @RequestParam(value="low_price") int lowPrice,
+            @RequestParam(value="high_price") int highPrice,
+            @RequestParam(value="number_of_people") int numberOfPeople,
+            @RequestParam(value="location") String location,
+            @RequestParam(value="destination") String destination,
+            @RequestParam(value="message") String message,
+            @RequestParam(value="pickup_time_timestamp") String pickUpTimeTimestamp
```

Comment:
when calling the endpoint to create new rider, you have to pass all of this data above, lines 47-54,
and the keys for the values is the variable `value`, so e.g. when sending the phone number when calling the endpoint, the key has to be `phone_number `